modules/retail/pages/retail-requirements.adoc

```diff
@@ -55,9 +59,22 @@ xref:installation:general-requirements.adoc[].
 |                          | [path]``/var/lib/Kiwi`` Minimum 10{nbsp}GB
 |===
 
-.Network Requirements
+
+
+== Network Requirements
 
 * The {susemgr} Server requires a reliable and fast WAN connection.
 * The branch server requires a reliable WAN connection, to reach the {susemgr} Server.
 * If you are using a dedicated network, the branch server requires at least two network interfaces: one connected to the WAN with reachable {susemgr} server, and one connected to the internal branch LAN.
 * Terminals require a LAN connection to the branch server network.
+
+
+
+== POS Terminal UEFI Secure Booting Requirements
+
+Secure boot from the network using UEFI PXE or UEFI HTTP is supported on both {sles}{nbsp}12 and {sles}{nbsp}15.
+UEFI Secure Boot is fully supported on {sles}{nbsp}15 images only.
+
+You cannot boot {sles}{nbsp}12 images using UEFI secure boot.
```

Comment:
This is missing critical piece of information - this concerns only local boot (that is boot from harddrive).

```suggestion
Booting from harddrive using UEFI Secure Boot is supported on {sles}{nbsp}15 images only.

You cannot boot {sles}{nbsp}12 images using UEFI secure boot from harddrive.
```